checks.d/zk.py

```diff
@@ -51,8 +105,12 @@ def check(self, instance):
         tags = instance.get('tags', [])
         cx_args = (host, port, timeout)
         sc_tags = ["host:{0}".format(host), "port:{0}".format(port)]
+        hostname = socket.gethostname()
```

Comment:
The preferred way to do this in the agent is via `get_hostname` in the `util` module. See [the mongo check](https://github.com/DataDog/dd-agent/blob/7312a61a6d359e461c3481e24bd98ae6e489306a/checks.d/mongo.py#L10) for an example